src/sparse_dd.ml

```diff
@@ -40,6 +40,10 @@ let set_machine_logging = ref false
 let experimental_reads_bypass_tapdisk = ref false
 let experimental_writes_bypass_tapdisk = ref false
 
+let ssl_legacy = ref true
```

Comment:
Set this to false initially; if a host has Host.ssl_legacy set to true then it should call sparse_dd with the relevant argument.